compiler/rustc_type_ir/src/ty_kind.rs

```diff
@@ -8,12 +9,15 @@ use rustc_data_structures::stable_hasher::{HashStable, StableHasher};
 use rustc_data_structures::unify::{NoError, UnifyKey, UnifyValue};
 #[cfg(feature = "nightly")]
 use rustc_macros::{Decodable, Encodable, HashStable_NoContext, TyDecodable, TyEncodable};
+#[cfg(feature = "nightly")]
+use rustc_serialize::Decodable;
```

Comment:
Inconsistent importing vs qualifying. Since you moved to qualifying in binder.rs, and mostly do it here, do it here fully, too